src/pages/recommendations.js

```diff
@@ -42,43 +48,83 @@ const Recommendations = ({setAuth}) => {
     }
   }
 
+  // dont know what this is right now??? --------------------
   useEffect(() => {
     getRecommendations();
```

Comment:
Must add a second argument of an empty array IF you only want to call getRecommendations() when the component gets created. Otherwise it'll get called every time it gets drawn.